walkthroughs/howto-k8s-tls-acm/README.md

```diff
@@ -0,0 +1,224 @@
+```bash
+export GREEN_CERTIFICATE_ARN=`aws acm request-certificate \
+    --domain-name "color-green.${SERVICES_DOMAIN}" \
+    --certificate-authority-arn ${ROOT_CA_ARN} \
+    --query CertificateArn --output text`
+```
+
+With managed certificates, ACM will automatically renew certificates that are nearing the end of their validity, and App Mesh will automatically distribute the renewed certificates. See [Managed Renewal and Deployment](https://aws.amazon.com/certificate-manager/faqs/#Managed_Renewal_and_Deployment) for more information.
+
+## Step 3: Create a Mesh with TLS enabled
+
+We are going to setup a mesh with four virtual nodes: Frontend, Blue, Green and Red, one virtual service: color and one virtual router: color.
+
+Let's create the mesh.
+
+```bash
+./mesh.sh up
+```
+
+Frontend has backend virtual service (color) configured and the virtual service (color) uses virtual router (color) as the provider. The virtual router (color) has three routes configured:
+- color-route-blue: matches on HTTP header "blue" to route traffic to virtual node `blue`
+- color-route-green: matches on HTTP header "green" to route traffic to virtual node `green` 
+- color-route-red: matches on HTTP header "red" to route traffic to virtual node `red`
+
+Virtual nodes `blue` and `green` are configured with TLS enabled for their respective listeners. Here's the spec for `green` Virtual Node:
+
+```
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualNode
+metadata:
+  name: green
+  namespace: howto-k8s-tls-file-based
+spec:
+  podSelector:
+    matchLabels:
+      app: color
+      version: green
+  listeners:
+    - portMapping:
+        port: 8080
+        protocol: http
+      healthCheck:
+        protocol: http
+        path: '/ping'
+        healthyThreshold: 2
+        unhealthyThreshold: 2
+        timeoutMillis: 2000
+        intervalMillis: 5000
+      tls:
+        mode: STRICT
+        certificate:
+          acm:
+            certificateARN: arn:aws:acm:us-west-2:<ACCOUNT_ID>>:certificate/<certificate>
+  serviceDiscovery:
+    dns:
+      hostname: color-green.howto-k8s-tls-file-based.svc.cluster.local
+```
+
+The `tls` block specifies the ACM certificate to use. See [Transport Layer Security (TLS)](https://docs.aws.amazon.com/app-mesh/latest/userguide/tls.html) for the IAM permissions that are required while using ACM PCA for TLS.
+
+
+## Setup 4: Verify TLS is enabled
+
+```bash
+kubectl -n default run -it --rm curler --image=tutum/curl /bin/bash
+```
+
+```
+curl -H "color_header: blue" front.howto-k8s-tls-file-based.svc.cluster.local:8080/; echo;
+
+curl -H "color_header: green" front.howto-k8s-tls-file-based.svc.cluster.local:8080/; echo;
+```
+
+You should see a successful response when using the HTTP header "color_header: blue"
+
+Let's check the SSL handshake statistics.
+
+```bash
+BLUE_POD=$(kubectl get pod -l "version=blue" -n howto-k8s-tls-file-based --output=jsonpath={.items..metadata.name})
```

Comment:
'file-based' here as well as the following line

Reply:
Updated to 'howto-k8s-tls-acm'